addons/project/i18n/project.pot

```diff
@@ -1931,6 +1930,22 @@ msgid ""
 "(parent_model) and task (model))"
 msgstr ""
 
+#. module: project
+#: model:ir.model.fields,help:project.field_project_project__privacy_visibility
+msgid ""
+"People to whom this project and its tasks will be visible.\n"
+"\n"
+"- Invited internal users: internal users can access the project and the tasks they are following. This access can be modified on each task individually by adding or removing the internal user as follower.\n"
+"A user with the project > administrator access right level can still access this project and its tasks, even if they are not explicitly part of the followers.\n"
+"\n"
+"- All internal users: all internal users can access the project and all of its tasks without distinction.\n"
+"\n"
+"- Invited portal users and all internal users: all internal users can access the project and all of its tasks without distinction.\n"
+"Portal users can only access the projects and the tasks they are following. This access can be modified on each task individually by adding or removing the portal user as follower.\n"
+"\n"
+"In any case, an internal user with no project access rights can still access a task, provided that he is given the corresponding URL (and that he is part of the followers if the project is private)."
+msgstr ""
+
```

Comment:
Ideally, in order to keep the diff minimal, I would not change the place of this traduction in the file.